release-notes/opensearch-ml-common.release-notes-2.16.0.0.md

```diff
@@ -0,0 +1,49 @@
+## Version 2.16.0.0 Release Notes
+
+Compatible with OpenSearch 2.16.0
+
+### Features
+* Add initial MLInferenceSearchResponseProcessor (#2688)[https://github.com/opensearch-project/ml-commons/pull/2688]
+* Add initial search request inference processor (#2731)[https://github.com/opensearch-project/ml-commons/pull/2731]
+* Feat: add bedrock runtime agent for knowledge base (#2651)[https://github.com/opensearch-project/ml-commons/pull/2651]
```

Comment:
I don't think this should be treated as features, may be enhancement?

Reply:
+1, let's move to enhancement. Also let's make sure the commit message is accurate when review PR in future. This message is quite misleading